Age-Calculator-GUI/age_calc_gui.py

```diff
@@ -0,0 +1,54 @@
+# import libraries
+from tkinter import *
+from datetime import date
+
+# initialized window
+root = Tk()
+root.geometry('280x300')
+root.resizable(0,0)
+root.title('Age Calculator')
+
+# defining the function for calculating age
+def ageCalc():
+    today = date.today()
+    birthDate = date(int(yearEntry.get()), int(monthEntry.get()), int(dayEntry.get()))
+    age = today.year - birthDate.year - ((today.day, today.month) < (birthDate.day, birthDate.month))
+    statement = Label(text=f"{nameValue.get()}'s age is {age}.")
+    statement.grid(row=6, column=1, pady=15)
+
```

Comment:
```suggestion
root.resizable(0,0)
root.title('Age Calculator')
statement = Label(root)

# defining the function for calculating age
def ageCalc():
    global statement
    statement.destroy()
    today = date.today()
    birthDate = date(int(yearEntry.get()), int(monthEntry.get()), int(dayEntry.get()))
    age = today.year - birthDate.year - ((today.day, today.month) < (birthDate.day, birthDate.month))
    statement = Label(text=f"{nameValue.get()}'s age is {age}.")
    statement.grid(row=6, column=1, pady=15)

```

Reply:
@mehabhalodiya Try this

Reply:
@santushtisharma10 Works well!
If we change one of the fields like name here, so it is working fine now like no statement error is there!

But the error of age calculation is still there. e.g. year = 2002, month = 1, day = 10
The answer should be 19 but here it's not working !

Something is missing like considering leap year calculation, maybe. Right?
Can you tell me what should I add in `age` formula?

Reply:
```suggestion
root.resizable(0,0)
root.title('Age Calculator')

# defining the function for calculating age
def ageCalc():
    today = date.today()
    birthDate = date(int(yearEntry.get()), int(monthEntry.get()), int(dayEntry.get()))
    age = today.year - birthDate.year
    if today.month < birthDate.month or today.month == birthDate.month and today.day < birthDate.day : age -= 1 
    statement = Label(text=f"{nameValue.get()}'s age is {age}.")
    statement.grid(row=6, column=1, pady=15)

```

@mehabhalodiya try this

Reply:
Now it's working!

In case of: year = 2002, month = 1, day = 10
The answer is 19!

Thank you @santushtisharma10 for helping me out!

Reply:
I am modifying it now!